test/trento/domain/cluster/cluster_test.exs

```diff
@@ -116,25 +116,165 @@ defmodule Trento.ClusterTest do
       )
     end
 
-    test "should return an error if the cluster was not registered yet and a command from a non-DC is received" do
-      assert_error(
-        [],
+    test "should add a host the cluster when the cluster has empty/unknown details" do
```

Comment:
shouldn't we  keep (in the title)  a reference to the fact that the command is from a non-dc?

Reply:
should be done in the @arbulu89 comments